tests/test_attach_cve_flaws.py

```diff
@@ -1,312 +0,0 @@
-import unittest
```

Comment:
Move tests from here to test_bzutil.py since that's the logical place.
In future we should split bzutl up since it's growing to be huge.